lib/init.go

```diff
@@ -0,0 +1,221 @@
+package lib
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/alecthomas/template"
+	"github.com/resumic/schema/docs/schema"
+	"github.com/tidwall/gjson"
+)
+
+//InitResume initializes the resume.json file
+func InitResume() {
+	var conf, doc, tmpl string
+	flag.StringVar(&conf, "conf", "../schema/schema.json", "Schema json file")
+	flag.StringVar(&tmpl, "tmpl", "resume.tmpl", "Template file")
+	flag.StringVar(&doc, "doc", "resume.json", "Output json file")
+	content, err := ioutil.ReadFile(conf)
```

Comment:
I think it's better to rename content to config or something else to avoid any confusion.